core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -1573,6 +1573,8 @@ class TaskSetManagerSuite
   }
 
   test("SPARK-21563 context's added jars shouldn't change mid-TaskSet") {
+    val jarPath = Thread.currentThread().getContextClassLoader.getResource("TestUDTF.jar")
```

Comment:
Do we need `testJar` too?

Reply:
I actually used `testJar` to avoid the name conflict with `jarPath`. This test case seems fine.

Reply:
I'm a bit confused.
`val testJar = Thread.currentThread().getContextClassLoader.getResource("TestUDTF.jar")` and val jarPath = `Thread.currentThread().getContextClassLoader.getResource("TestUDTF.jar")` both exists.

Reply:
I see. But both names are already being used here and there though ... I used `testJar` because of:

```
val jarPath = Thread.currentThread().getContextClassLoader.getResource("TestUDTF.jar").toString
```

Otherwise, it fails with `NullpointerException`.

Reply:
I originally thought you want unify the name of `testJar` and `jarPath`. Now I got it. Some tests cases need the null check for `testJar` before `val jarPath = testJar. toString`.